lib/src/components/NxSearchDropdown/NxSearchDropdown.scss

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-container-helpers';
+
+.nx-search-dropdown {
+  display: inline-block;
+  position: relative;
+
+  &__menu {
+    // only show when the component is focused
```

Comment:
This isn't true, it only shows when it's focused and the user has typed something

Reply:
> This isn't true, it only shows when it's focused and the user has typed something

What you said is a subset of what I said.  But only the "when focused" part of that is implemented in this spot.